app/src/citations.py

```diff
@@ -15,7 +15,7 @@
 def get_context_for_prompt(chunks: Sequence[ChunkWithScore]) -> str:
     return "\n\n".join(
         [
-            f"Citation: chunk-{index}\nDocument name: {chunk.chunk.document.name}\nContent: {chunk.chunk.content}"
+            f"Citation: chunk-{index}\nDocument name: {chunk.chunk.document.name}\nContent: {chunk.chunk.content}\nHeadings: {chunk.chunk.headings}"
```

Comment:
I think it would make more sense to put the headings before the content rather than after. Not sure if it'll have too big a difference on LLM performance, but that's at least what a human would find intuitive.